lib/domain/dtos/UpdateFlpDto.js

```diff
@@ -17,6 +17,7 @@ const EntityIdDto = require('./EntityIdDto');
 const OptionalPositiveIntegerDto = Joi.number()
     .integer()
     .positive()
```

Comment:
You can replace `.positive().allow(0)` with `min(0)`

Reply:
Isn't it the same?

Reply:
Yes, but in one line 😄 
Not a blocker comment though